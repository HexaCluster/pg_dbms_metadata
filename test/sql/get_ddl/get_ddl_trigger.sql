SELECT dbms_metadata.get_ddl('TRIGGER','trigger_double_salary','gdmmm');