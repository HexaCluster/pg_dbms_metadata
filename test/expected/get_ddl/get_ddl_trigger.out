CREATE TRIGGER "order" BEFORE INSERT ON gdmmm.table_all FOR EACH ROW EXECUTE FUNCTION gdmmm.double_salary()
